electron/src/lib/CoreProtocol.test.main.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from 'assert';
+import sinon from 'sinon';
+import {CustomProtocolHandler} from './CoreProtocol';
+import {EVENT_TYPE} from "./eventType";
+
+let protocolHandler: CustomProtocolHandler = undefined;
+
+describe('dispatcher', () => {
```

Comment:
One last thing, let's rename this function `dispatch` instead of `dispatcher`. WDYT?

Reply:
I would give it an even more precise name, like `dispatchDeepLink`.

Reply:
Sure :+1: